src/module.ts

```diff
@@ -374,39 +372,33 @@ export default defineNuxtModule<ModuleOptions>({
       return
     }
 
-    // Create storage instance
-    const storage = createStorage()
-    const mounts = useContentMounts(nuxt, contentContext.sources || [])
-    for (const mount in mounts) {
-      storage.mount(mount, getMountDriver(mounts[mount]))
-    }
-
-    const ws = createWebSocket()
+    nuxt.hook('nitro:init', (nitro) => {
+      const ws = createWebSocket()
 
-    // Dispose storage on nuxt close
-    nuxt.hook('close', async () => {
-      await Promise.all([
-        storage.dispose(),
-        ws.close()
-      ])
-    })
+      // Dispose storage on nuxt close
+      nuxt.hook('close', async () => {
```

Comment:
i think you can also use `nitro.hooks.hook('close')` since we are in nitro context and was depends on nitro storage